Shopipy.ApiService/Services/AuthService.cs

```diff
@@ -0,0 +1,30 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using Shopipy.Persistence.Models;
+
+namespace Shopipy.ApiService.Services;
+
+public class AuthService(IConfiguration config)
```

Comment:
Try to pass an option instead of the whole configuration.

Reply:
Why do you want that?

Reply:
So its easier to mock options and test the service working correctly.

Reply:
Changed